modules/packer/custom-image/image.pkr.hcl

```diff
@@ -197,4 +201,25 @@ build {
       "echo \"Image built: $(jq -r '.builds[-1].artifact_id' ${var.manifest_file} | cut -d ':' -f2)\"",
     ]
   }
+
+  # If there is an error during image creation, print out command for getting packer VM logs
+  error-cleanup-provisioner "shell-local" {
+    environment_vars = [
+      "PRJ_ID=${var.project_id}",
+      "INST_NAME=${local.instance_name}",
+      "ZONE=${var.zone}",
+    ]
+    inline = [
+      "#!/bin/bash -e",
```

Comment:
I think you want to use `execute_command` parametmer and `set -e` inside the script. The `provisioner` block above is an example (I believe  error-cleanup-provisioner implements all the parameters)

https://developer.hashicorp.com/packer/docs/provisioners/shell-local#execute_command

Reply:
The execute command is specifically for scripts that are linked, I believe.  I've added the `inline_shebang` to specify the shell command.